src/main/resources/application.yml

```diff
@@ -1,22 +1,24 @@
 server:
-  port: 8090
+  port: ${SERVER_PORT:8090}
 spring:
   application:
     name: eu-interop-federation-gateway
   datasource:
-    driver-class-name: org.h2.Driver
```

Comment:
Keep the H2 settings and overwrite them with Docker

Reply:
Please, check my other response. H2 should only be for testing so I changed H2 configuration to the testing scope (src/test/resources).